phys2bids/io.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
```

Comment:
Just noticed this. There's no need for a shebang for files in the package, unless they will be run from the command line (and even then, if they're set up as CLIs there's probably no need). See [here](https://stackoverflow.com/a/19305076/2589328).

Reply:
It's true, it's not necessary. Since it doesn't hurt anyone though (and AFAIK it's not against pep8), let's leave it there - it helps (some) editors and (some) humans to know it's python. (You might argue the extension should be enough, but still.)